packages/waku/src/lib/middleware/dev-server-impl.ts

```diff
@@ -29,6 +29,7 @@ import { rscDelegatePlugin } from '../plugins/vite-plugin-rsc-delegate.js';
 import type { ClonableModuleNode, Middleware } from './types.js';
 import { fsRouterTypegenPlugin } from '../plugins/vite-plugin-fs-router-typegen.js';
 import { hackTailwindcss4Stackblitz } from '../plugins/hack-tailwindcss4-stackblitz.js';
+import type { AddressInfo } from 'node:net';
```

Comment:
I think we prefer not to directly use node.js API here

Reply:
The file already pulls in `node:stream` and `node:http` up at the top of the imports (I could probably move this import to be up next to those too)

Reply:
oh I didn't see that.